### DISCUSSION TITLE
Queueing up syncs for multiple commits in quick succession

### DISCUSSION THREAD

**Discussion by lacarvalho91** [argoproj/argo-cd#8375] [General]
Hi,

For my use-case I need ArgoCD to sync every commit, in the order they are made. I am using webhooks to trigger refreshes and I have `timeout.reconciliation` set to 0 to disable polling.

From the testing I am doing it looks like if I make a commit `A`, and then commit `B` and `C` whilst `A` is still syncing, `B` will never get sync'd and ArgoCD will skip straight to `C` after `A` has finished syncing.

Is this expected? Is it possible to get the desired behaviour?

**Comment by jurgenweber** [comment]
Did anyone find a solution to this outside of this discussion?
